apps/condo/domains/common/constants/fileExtensions.js

```diff
@@ -0,0 +1,5 @@
+export const TABLE_UPLOAD_ACCEPT_FILES = [
```

Comment:
I think really this const will be used only in one place. And maybe it's better to place it near the usage (Optional)